masterlist.yaml

```
@@ -13983,6 +13983,10 @@ plugins:
       - 'CoW Entry Requirements.esp'
       - 'NotSoFast-MageGuild.esp'
       - 'MCWT_TheAugursAdmonishment.esp'
+    msg:
+      - <<: *patchProvided
+        subs: ['College of Winterhold - Quest Expansion']
+        condition: 'active("At Your Own Pace - College of Winterhold.esp") and not active("At Your Own Pace - College Quest Expansion Patch.esp")'
```

The condition needs to be adjusted as well:

```suggestion
        condition: 'active("College Of Winterhold - Quest Expansion.esp") and not active("At Your Own Pace - College Quest Expansion Patch.esp")'
```